src/Microsoft.DotNet.Watcher.Tools/Microsoft.DotNet.Watcher.Tools.csproj

```diff
@@ -20,7 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Process.Sources" Version="1.2.0-*" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="$(AspNetCoreVersion)" />
```

Comment:
The 1.1.0-preview4-final (project.json) version of tools use API from M.Ext.CommandLineUtils that was in the 1.1.0 release, but was missing from 1.0.0. Because the MSBuild fork of this repo had to roll back from 1.1.0 dependencies to 1.0.0, we copied the source code here. We would have used the .Sources package, but the pre-release version of the MSBuild SDK we used didn't support .Source packages at the time. (Sidenote: the rel/1.0.0 branch of this project is still on an early pre-release SDK. It won't open in VS 2017

 This PR upgrades to 2.0 dependencies which means we can revert to using a package reference.